app/src/main/java/ru/practicum/android/diploma/favorites/data/converter/FavoriteVacancyDbConverter.kt

```diff
@@ -0,0 +1,53 @@
+package ru.practicum.android.diploma.favorites.data.converter
+
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
+import ru.practicum.android.diploma.common.domain.model.VacancyDetails
+import ru.practicum.android.diploma.common.domain.model.VacancyFromList
+import ru.practicum.android.diploma.favorites.data.entity.FavoriteVacancyEntity
+
+class FavoriteVacancyDbConverter : KoinComponent {
```

Comment:
здесь можно внедрить gson через коин-модуль напрямую в конструктор, т.е. лучше напиши
```suggestion
class FavoriteVacancyDbConverter (gson: Gson) {
```
и не нужно будет наследовать класс от KoinComponent и прописывать val в теле класса